sys-kernel/buildkernel/buildkernel-1.0.37.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2020 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
 inherit eutils
 
 DESCRIPTION="Build secure boot EFI kernel with LUKS, LVM and plymouth"
-BASE_SERVER_URI="https://github.com/sakaki-"
+BASE_SERVER_URI="https://github.com/GenPi64"
```

Comment:
:thinking: this way the ebuild can not install any more:
```
>>> Downloading 'https://github.com/GenPi64/buildkernel/releases/download/1.0.37/buildkernel-1.0.37.tar.gz'
--2023-02-13 22:38:22--  https://github.com/GenPi64/buildkernel/releases/download/1.0.37/buildkernel-1.0.37.tar.gz
```
https://github.com/sakaki-/buildkernel/ was never forked here.

Reply:
Well wait, what's even including this? We don't use this ebuild in our image builder in the first place?

Reply:
Some out of date Sakaki builds probably that shouldn't be using it anymore, but it's a dep of genup I think.